pytorch/training/docker/1.8/py3/cu111/Dockerfile.gpu

```diff
@@ -290,6 +290,11 @@ RUN SMDATAPARALLEL_PT=1 pip install --no-cache-dir ${SMDATAPARALLEL_BINARY}
 
 ENV LD_LIBRARY_PATH="/opt/conda/lib/python3.6/site-packages/smdistributed/dataparallel/lib:$LD_LIBRARY_PATH"
 
+# Install PT S3 plugin
+ARG PT_S3_WHL_GPU=https://aws-s3-plugin.s3-us-west-2.amazonaws.com/binaries/0.0.1/93fdaed/awsio-0.0.1-cp36-cp36m-manylinux1_x86_64.whl
+RUN pip install --no-cache-dir -U ${PT_S3_WHL_GPU}
+RUN mkdir -p /etc/pki/tls/certs && cp /etc/ssl/certs/ca-certificates.crt /etc/pki/tls/certs/ca-bundle.crt
```

Comment:
Are you sure it's okay to copy the certificates rather than sym-linking to them?